dist/Dockerfile.deploy/Dockerfile

```diff
@@ -1,14 +1,10 @@
-FROM registry.centos.org/centos/centos:7 as builder
+FROM redhat/ubi8:latest as builder
```

Comment:
What is `redhat/ubi8:latest` supposed to resolve to? This should be either just `ubi8` or its long form `registry.access.redhat.com/ubi8`. If I just do `podman build . -f dist/Dockerfile.deploy/Dockerfile` with the file with `redhat/ubi8:latest` then podman asks me what to resolve to and it treats `redhat` as a repository in =e.g. quay:

```
$ podman build . -f dist/Dockerfile.deploy/Dockerfile
[1/2] STEP 1/10: FROM redhat/ubi8:latest AS builder
? Please select an image: 
    registry.fedoraproject.org/redhat/ubi8:latest
    registry.access.redhat.com/redhat/ubi8:latest
  ▸ docker.io/redhat/ubi8:latest
    quay.io/redhat/ubi8:latest
```

With `FROM registry.access.redhat.com/ubi8:latest` the build goes just fine.

Reply:
fixed it to use quay